redisinsight/ui/src/components/new-index/create-index-step/CreateIndexStepWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
```

Comment:
Just checking - was `CreateIndexStepWrapper` intentionally added to this PR?

Reply:
Good question - initially, I started the work on this component, based on the branch related to the `Tabs` component, with the idea to add the `Field Boxes` inside the "Use Preset Index" tab. Later on, I gave up on this idea and provided this PR, with the component being stand-alone. 

So, for safety’s sake, I just rebased the branch with the latest state of `feature/RI-6855/vector-search`, and I hope we're safe to move on with this one.